src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -211,7 +217,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- TODO perform any preprocess transforms on the files -->
 
     <ItemGroup>
-      <ResolvedAssembliesToPublish Include="@(ReferenceCopyLocalPaths)" Exclude="@(ResolvedAssembliesToPublish)">
+      <ResolvedAssembliesToPublish Include="@(ReferenceCopyLocalPaths)"
+                                   Exclude="@(ResolvedAssembliesToPublish)"
+                                   Condition="'$(PublishProjectReferenceDocumentationFiles)' == 'true' or '%(Extension)' != '.xml'">
```

Comment:
@rainersigwald @cdmihai @KirillOsenkov @jeffkl - are documentation files the only type of `xml` files that can get put in `@(ReferenceCopyLocalPaths)`?  Is there some other way to distinguish what is a P2P reference's doc file?

I'm not sure who else would know this, maybe @ericstj @weshaggard @davkean ...?